### PR TITLE
245 UI fix  fix the rating part in full recipe display  description of the recipe

### PR DESCRIPTION
**Pull request by JeannedeM**
Tasks :

- [x]    Add description of recipe to full display.
- [x]     Fix rating position on recipe full display.
- [x]     Fix user name position on full display.

### CODE REVIEW

File: app/src/main/java/com/android/feedme/ui/component/RecipeFullDisplay.kt [hunk 1]
```diff
@@ -201,17 +201,18 @@ fun GeneralInfoDisplay(
...
+                            profileViewModel.setViewingProfile(profile)
+                            navigationActions.navigateTo(Screen.PROFILE)
+                          }),
+              text = "@${profile.username}",
               textAlign = TextAlign.Center,
               color = BlueUsername,
               style = TextStyle(fontSize = 16.sp, fontWeight = FontWeight.Medium))
         }
-        Spacer(modifier = Modifier.weight(1f))
+        // Spacer(modifier = Modifier.weight(1f))
```
Comment (ohamz) [inline app/src/main/java/com/android/feedme/ui/component/RecipeFullDisplay.kt:215]: I find the username and the rating really close, why did you remove this spacer ?

image

Comment (ohamz) [reply app/src/main/java/com/android/feedme/ui/component/RecipeFullDisplay.kt]: Yes I think it's better with the Spacer.


> people found a bit weird the way user name and rating was display.

Actually I think people found it weird when there is only the ratins displayed (when no author for the recipe is found), so maybe add the Spacer to the if statement of when the profile username is displayed ?

Comment (JeannedeM) [reply app/src/main/java/com/android/feedme/ui/component/RecipeFullDisplay.kt]: If there is no user name, the rating will be centered like this (i moved the spacer inside the if, like you said).
image

Comment (0marwa) [reply app/src/main/java/com/android/feedme/ui/component/RecipeFullDisplay.kt]: I think that's a good solution!

**Review by 0marwa — APPROVED** [review]
This fix seems to have cleaned up the whole username/rating display situation, good job! 
And the UI still matches the original Figma design with the description added. The code is clean, so good job (:
